README.md

```diff
@@ -13,6 +13,16 @@ See the [documentation](https://appsody.dev/docs) for more details:
 * [Installation](https://appsody.dev/docs/getting-started/installation)
 * [CLI Reference](https://appsody.dev/docs/using-appsody/cli-commands)
 
+For information on building the Appsody CLI from source, see [Building from Source](build.md)
 
+## Contributing
 
-For information on building the Appsody CLI from source, see [Building from Source](build.md)
\ No newline at end of file
+We welcome all contributions to the Appsody project. Please see our [Contributing guidelines](https://github.com/appsody/docs/blob/master/CONTRIBUTING.md)
+
```

Comment:
The docs repo has an additional line about "If you find a bug..."

Reply:
Yep, it would be good to add a similar thing to this repo, pointing to Appsody cli issues. will add this in